docs/02-guides/public-configuration.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Public configurations
+description:
+  The public configuration is a set of configuration options that are available
+  to any user of the application, they are accessible both from the client and
+  the server.
+sidebar_position: 5
+---
+
+<p>{frontMatter.description}</p>
+
+## Accessing Public Configuration
+
```

Comment:
**nitpick:** it'd be good to explain how to access them server side

**suggestion:** add a sentence mentioning the `config.public` namespace